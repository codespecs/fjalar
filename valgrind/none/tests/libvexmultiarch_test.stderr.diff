--- libvexmultiarch_test.stderr.exp	2022-08-03 11:30:53.833310523 -0700
+++ libvexmultiarch_test.stderr.out	2022-08-04 11:59:01.202903934 -0700
@@ -0,0 +1,2 @@
+valgrind: You cannot run './libvexmultiarch_test' directly.
+valgrind: You should use $prefix/bin/valgrind.
