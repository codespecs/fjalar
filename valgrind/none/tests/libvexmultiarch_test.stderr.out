valgrind: You cannot run './libvexmultiarch_test' directly.
valgrind: You should use $prefix/bin/valgrind.
